netci.groovy

```diff
@@ -200,15 +200,6 @@ def supportedFullCycleOuterloopPlatforms = ['Windows_NT', 'Ubuntu14.04', 'Ubuntu
                     }
                 }
             }
-            else if (isPR) {
-                newJob.with {
-                    steps {
-                        shell("HOME=\$WORKSPACE/tempHome ./build.sh -${configurationGroup.toLowerCase()}")
-                        shell("HOME=\$WORKSPACE/tempHome ./build-tests.sh -${configurationGroup.toLowerCase()} -outerloop -testWithLocalLibraries -- /p:OSGroup=${osGroupMap[os]} /p:ServiceUri=\$WcfServiceUri /p:SSL_Available=true /p:IsCIBuild=true")
-                    }
-                }
-            }
-            //Enable Certificate Tests on non PR runs first to ensure they are reliable, will enable on PR if we are green for one week
             else {
                 newJob.with {
                     steps {
```

Comment:
NIT: it would be helpful if we can have a comment explaining why `/p:Peer_Certificate_Installed=true` is not enforced for non-Windows platforms.